gateway/1.4/modules/ROOT/pages/flex-gateway-delete.adoc

```diff
@@ -2,4 +2,37 @@
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
-:imagesdir: ../assets/images
\ No newline at end of file
+:page-aliases: runtime-manager::flex-gateway-delete.adoc
+:imagesdir: ../assets/images
+
+To delete a gateway and its replicas in Runtime Manager:
+
+. From Anypoint Platform, select *Runtime Manager*.
+. Select *Flex Gateway* in the left menu.
```

Comment:
Try not to use "left" because it is non-accessible.

Reply:
Fixed